### PR TITLE
streamlit-mermaid 0.2.0 :snowflake:

### PR DESCRIPTION
**Pull request by tombenes**
streamlit-mermaid 0.2.0 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5227]
- dev_url:        https://github.com/neka-nat/streamlit-mermaid/tree/main
- conda_forge:    None
- pypi:           n/a
- pypi inspector: n/a

### Explanation of changes:

- new version number

[PKG-5227]: https://anaconda.atlassian.net/browse/PKG-5227?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,45 +1,51 @@
...
+  number: 0
 
 requirements:
   host:
     - python
+    - poetry-core
     - pip
   run:
     - python
+    - streamlit >=1.0
```
Comment (ifitchet) [inline recipe/meta.yaml]: `pyproject.toml` and `setup.py` both say `streamlit >= 1.24.1` ??

Comment (ifitchet) [reply recipe/meta.yaml]: Wait, lol, the pypi tarball and git repo are different...

Comment (tombenes) [reply recipe/meta.yaml]: This differs between tarball and github, do you think we should go with higher version just to be safe?

Comment (ifitchet) [reply recipe/meta.yaml]: If we could ping the authors and ask "why so?" that might be good.
Technically, as we're building from the pypi tarball, I'd follow its dependencies.
The git repo hasn't had those files touched, even if they've made other changes (like the LlCENSE file 😉 ).

Hmm, even the 0.1.0 pypi tarball has the higher dependency.

Plus, if it builds and tests within our ecosystem, then we should be OK.

Comment (tombenes) [reply recipe/meta.yaml]: I've updated the version to be as in tarball

**Review by psteyer — APPROVED** [review]

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,45 +1,51 @@
...
+{% set version = "0.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: <sha256>
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/streamlit-mermaid-{{ version }}.tar.gz
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:9]: FYI, a PyPI project leads to the wrong home URL https://github.com/AI-Yash/st-chat (but was fixed here https://github.com/neka-nat/streamlit-mermaid/commit/6eac163199b384bc249da06c417459a74f00a2d6).
Another question is that the upstream source on github has additional folders in `streamlit_mermaid/frontend` but the pypi tarball has only `streamlit_mermaid/frontend/build`.
I wonder if the package isn't vulnerable somehow but I have no idea how to verify it.

UPD. Here is why only `streamlit_mermaid/frontend/build` is packaged https://github.com/neka-nat/streamlit-mermaid/commit/324873e3a8602fc108b72ff1a6a61a010cd7ad72

**Review by skupr-anaconda — APPROVED** [review]
LGTM but the package project is badly managed by the maintainer

**Pull request merged**